pkg/addons/data/Deployment.apps/openshift-infra/dedicated-admin-reconciler.yaml

```diff
@@ -0,0 +1,105 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dedicated-admin-reconciler
+  namespace: openshift-infra
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dedicated-admin-reconciler
+  template:
+    metadata:
+      labels:
+        app: dedicated-admin-reconciler
+    spec:
+      serviceAccountName: ded-admin-reconciler
+      containers:
+      - args:
+        - |
+          set -o errexit
+          set -o nounset
+          set -o pipefail
+
+          cat <<'SCRIPT' >/tmp/rolebindings
+          kind: List
+          apiVersion: v1
+          metadata: {}
+          items:
+          - apiVersion: rbac.authorization.k8s.io/v1
+            kind: RoleBinding
+            metadata:
+              name: admin
+            roleRef:
+              apiGroup: rbac.authorization.k8s.io
+              kind: ClusterRole
+              name: admin
+            subjects:
+            - apiGroup: rbac.authorization.k8s.io
+              kind: Group
+              name: dedicated-admins
+            - apiGroup: rbac.authorization.k8s.io
+              kind: Group
+              name: system:serviceaccounts:dedicated-admin
+          - apiVersion: rbac.authorization.k8s.io/v1
+            kind: RoleBinding
+            metadata:
+              name: dedicated-project-admin
+            roleRef:
+              apiGroup: rbac.authorization.k8s.io
+              kind: ClusterRole
+              name: dedicated-project-admin
+            subjects:
+            - apiGroup: rbac.authorization.k8s.io
+              kind: Group
+              name: dedicated-admins
+            - apiGroup: rbac.authorization.k8s.io
+              kind: Group
+              name: system:serviceaccounts:dedicated-admin
+          - apiVersion: rbac.authorization.k8s.io/v1
+            kind: RoleBinding
+            metadata:
+              name: view
+            roleRef:
+              apiGroup: rbac.authorization.k8s.io
+              kind: ClusterRole
+              name: view
+            subjects:
+            - apiGroup: rbac.authorization.k8s.io
+              kind: Group
+              name: dedicated-readers
+            - apiGroup: rbac.authorization.k8s.io
+              kind: Group
+              name: system:serviceaccounts:dedicated-readers
+          ##### END HERE #######
+          SCRIPT
+
+          cat <<'SCRIPT' >/tmp/reconcile
+          #!/bin/bash -x
+          set -o errexit
+          set -o nounset
+          set -o pipefail
+
+          project=$1
+          echo "Project: ${project}"
+
+          # do nothing for whitelisted projects
+          for ns in 'default' 'kube-' 'openshift';
+          do
+            if [ -z "${project##$ns*}" ]; then
+              exit 0
+            fi
+          done
+
+          oc auth reconcile -f /tmp/rolebindings -n"${project}"
```

Comment:
nit: add a space between -n and "${project}"

Reply:
fixed.